manual-install/update-yaml.sh

```diff
@@ -41,6 +42,7 @@ sed -i '/AIO_TOKEN/d' containers.yml
 sed -i '/AIO_URL/d' containers.yml
 sed -i '/DOCKER_SOCKET_PROXY_ENABLED/d' containers.yml
 sed -i '/ADDITIONAL_TRUSTED_PROXY/d' containers.yml
+sed -i 's/\(^[[:space:]]*- \([^=]*\)\)=%\2%/\1/' containers.yml
```

Comment:
```suggestion
sed -ie 's/\( *- \(\w*\)\)=\${\2\}/\1/' containers.yml
```

I don't really understand the purpose of this line... If it's for environment variables, this regex seems better.

```diff
-      - TURN_SECRET=${TURN_SECRET}
+      - TURN_SECRET
-      - SIGNALING_SECRET=${SIGNALING_SECRET}
+      - SIGNALING_SECRET
-      - ONLYOFFICE_SECRET=${ONLYOFFICE_SECRET}
+      - ONLYOFFICE_SECRET
```

Test online: https://sed.js.org/